packages/utils/test-tools/src/enzymeTests.tsx

```diff
@@ -86,3 +88,51 @@ export function checkReRender(render: JSXProducer, depth: number = 1) {
   const t2 = snapshotPropTree(w2, depth, filter);
   compareTrees(t1, t2, []);
 }
+
+/**
+ * Validate that value(s) returned by a hook do not change in identity.
+ * @param testDescription - Custom test description
+ * @param useHook - Function to invoke the hook and return an array of return values which
+ * should not change
+ * @param useHookAgain - If you want to verify that the return value doesn't change when hook
+ * parameters change, you can pass this second callback which calls the hook differently.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function validateHookValueNotChanged<TValues extends NonNullable<any>[]>(
+  testDescription: string,
+  useHook: () => TValues,
+  useHookAgain?: () => TValues,
+) {
+  it(testDescription || 'returns the same value(s) each time', () => {
+    let latestValues: TValues | undefined;
+    let callCount = 0;
+
+    const TestComponent: React.FunctionComponent = () => {
+      callCount++;
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      latestValues = callCount === 1 ? useHook() : (useHookAgain || useHook)();
+      return <div />;
+    };
```

Comment:
Feels iffy we're conditionally calling a hook. Was this also copied from FluentUI React?

Reply:
Also, return a `<View>`, not a `<div>`?

Reply:
Yes, the test was copied from here:

https://github.com/microsoft/fluentui/blob/6a2ebfd797c041225db94e6550e41e0e59672fb7/packages/react-hooks/src/testUtilities.tsx#L13

I changed the <div> for a <View>

Reply:
I'll resolve this then. I imagine because it's a test on the hook, it has to do weird stuff